gstd/src/msg/basic.rs

```diff
@@ -251,10 +220,27 @@ pub fn reply_bytes<T: AsRef<[u8]>>(payload: T, value: u128) -> Result<MessageId>
 /// # See also
 ///
 /// [`reply_push`] function allows to form a reply message in parts.
+#[wait_for_reply]
 pub fn reply_with_gas(payload: &[u8], gas_limit: u64, value: u128) -> Result<MessageId> {
     gcore::msg::reply_with_gas(payload, gas_limit, value).into_contract_result()
 }
 
+/// Same as [`replay`], without encoding payload.
```

Comment:
```suggestion
/// Same as [`reply`], without encoding payload.
```

Reply:
> Should we also specify here path to this fn? `(crate::msg::basic::reply)`

Need to try building docs, sometimes it finds adjacent functions automatically.